public/data/ssensalo.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "Edgar Ssensalo",
+    "bio": "Software Engineer & Machine Learning Pro",
+    "avatar": "https://github.com/ssensalo.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/ssensalo",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/SsensaloEdgar",
+        "icon": "twitter"
+      },
+      {
+        "name": "Connect with me on LinkedIn",
+        "url": "https://www.linkedin.com/in/edgar-ssensalo-572600167/",
+        "icon": "linkedin"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Started Freelancing",
+        "date": "March 2021",
+        "icon": "dollar",
+        "color": "green",
+        "description": "Started my freelancing journey",
+        "url": "https://www.edgarssensalo.io/"
+      }
+    ]
+}
```

Comment:
```suggestion
}

```

Reply:
@rohan-kulkarni-25 said I should remove that if you check the first suggested changes in the first commit